zookeeper-server/src/main/java/org/apache/zookeeper/server/PrepRequestProcessor.java

```diff
@@ -0,0 +1,1156 @@
+/*
```

Comment:
### ℹ Getting worse: Primitive Obsession
The ratio of primitive types in function arguments increases from 45,16% to 50,00%, threshold = 30,0%